high_card.rb

```diff
@@ -1,5 +1,51 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
```

Comment:
I think there would be a way to hard-code a rule but this seems more efficient - unless, of course, you're suggesting there's an alternate to that.